### PR TITLE
Add a prerelease workflow to the "next" branch

### PR DESCRIPTION
**Pull request by Westbrook**
## What I did

1. Set a workflow that publishes prereleases on `next`, I think...

### CODE REVIEW

**Comment by Westbrook** [comment]
@bennypowers, I pulled out the blog sync, as it seemed overkill, but other wise this seems like it's in a good place to test out?

**Review by bennypowers — APPROVED** [review]

File: .husky/_/husky.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```
Comment (abdonrd) [inline .husky/_/husky.sh:1]: Why this file? Should be ignored, right?

File: packages/dev-server-storybook/package.json [hunk 2]
```diff
@@ -63,7 +63,7 @@
     "@web/dev-server-core": "^0.3.0",
     "@web/rollup-plugin-html": "^1.3.2",
     "@web/rollup-plugin-polyfills-loader": "^1.0.3",
-    "@web/storybook-prebuilt": "^0.1.20",
+    "@web/storybook-prebuilt": "^0.1.22",
```
Comment (abdonrd) [inline packages/dev-server-storybook/package.json:66]: There is a new `0.1.23`

File: .husky/_/husky.sh (hunk 1 above)
Comment (Westbrook) [reply .husky/_/husky.sh]: Thanks!

File: packages/dev-server-storybook/package.json (hunk 2 above)
Comment (Westbrook) [reply packages/dev-server-storybook/package.json]: Updated to latest alpha...

File: .github/workflows/canary.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@master
```
Comment (web-padawan) [inline .github/workflows/canary.yml:23]:
```suggestion
        uses: actions/setup-node@v2
```

File: .github/workflows/canary.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+jobs:
+  release:
+    # Prevents changesets action from creating a PR on forks
+    if: github.repository == 'modernweb-dev/web'
+    name: Pre-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
```
Comment (web-padawan) [inline .github/workflows/canary.yml:17]:
```suggestion
        uses: actions/checkout@v2
```

File: .github/workflows/canary.yml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+          registry-url: 'https://registry.npmjs.org'
```
Comment (web-padawan) [inline .github/workflows/canary.yml:26]: Is this needed?

File: .github/workflows/canary.yml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
```
Comment (web-padawan) [inline .github/workflows/canary.yml:38]: Let's use the built-in cache, see #1586

File: .github/workflows/canary.yml (hunk 5 above)
Comment (Westbrook) [reply .github/workflows/canary.yml:26]: Looks like it defaults to GPR (maybe). I grabbed this from https://github.com/modernweb-dev/web/blob/master/.github/workflows/release.yml#L26 originally...

File: .github/workflows/canary.yml [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js 12
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
```
Comment (web-padawan) [inline .github/workflows/canary.yml:25]: Please add `cache : 'yarn'` to enable cache, see #1586

**Review by bennypowers — APPROVED** [review]
LGTM. the loss of genuine 'prerelease' flow is lamentable, but kind of out of our hands, barring work on changesets itself.

**Review by web-padawan — APPROVED** [review]

**Pull request merged**